cfpb/cfpb-chart-builder
# Tile map responsive css

Update the tile map to use responsive CSS and media queries like the line charts and bar charts do!

## Additions

- styles and classes for font + positioning on the tile map legend + labels
- added some variables for the colors used in the tile map JS for easier maintenance

## Removals

- responsive Highcharts options since these are no longer needed 

## Changes

- fix chart heading styles so the text wraps on narrow screens
- use `renderer.label` instead of `text()` method to render the legend labels
- updated renderer methods to append legend `rect` and `label` elements inside an svg `g` group element for easier targeting as a group with CSS

## Testing

- check out this branch
- run `./setup.sh`
- run `gulp watch` and look scroll down to tile map

## Review

- @mistergone or @anselmbradford or @sebworks whoever will claim this first

[Preview this PR without the whitespace changes](?w=0)

## Screenshots
![screen shot 2017-06-13 at 3 14 02 pm](https://user-images.githubusercontent.com/702526/27105447-9cfa9122-505d-11e7-977a-64c7425cd98f.png)


![screen shot 2017-06-13 at 5 23 03 pm](https://user-images.githubusercontent.com/702526/27105441-97815604-505d-11e7-89cd-f838b344f0ee.png)


## Notes

-

## Todos

-

## Checklist
